apps/aecore/test/aec_fork_signalling_tests.erl

```diff
@@ -0,0 +1,130 @@
+%%%============================================================================
+%%% @copyright (C) 2019, Aeternity Anstalt
+%%% @doc
+%%% EUnit tests for aec_fork_signalling.
+%%% @end
+%%%============================================================================
+-module(aec_fork_signalling_tests).
+
+-include_lib("eunit/include/eunit.hrl").
+-include_lib("aecore/include/blocks.hrl").
+
+-define(TEST_MODULE, aec_fork_signalling).
+
+-define(SIGNALLING_START_HEIGHT, 2).
+-define(SIGNALLING_END_HEIGHT, 5).
+-define(SIGNALLING_BLOCK_COUNT, 2).
+
+-define(FORK_HEIGHT, 7).
+
+-define(INFO_FIELD_AGAINST, make_info(0)).
+-define(INFO_FIELD_SUPPORT, make_info(1)).
+-define(INFO_FIELD_OTHER, make_info(999)).
+
+-define(VERSION_OLD, 4).
+-define(VERSION_NEW, 5).
+
+%% [0] --- [1] --- [2] --- [3] --- [4] --- [5] --- [6] --- [7] ---
+%%         H1      HS            HE - 1    HE    H2 - 1    H2
+
+-define(FORK_CFG,
```

Comment:
Shall this value be passed through the same validation function as used for the user config - as a sanity check that the test data is meaningful? Though it may be considered as breaking the boundaries of the unit test, so debatable, so optional.